setup.py

```diff
@@ -27,7 +28,7 @@
     assert False, "Error: Could not open '%s' due %s\n" % (filepath, error)
 
 INSTALL_REQUIRE = [
-    "torch>=1.11",
+    "torch>=1.11,<2.4" if sys.platform.startswith("win") else "torch>=1.11",
```

Comment:
Would prefer to have a warning requiring to downgrade torch to 2.3 or update VS C++  for the impacted architectures instead of pinning torch, would that work for you @helena-intel ?